website/api/zkvm/zkvm_overview.md

```diff
@@ -99,10 +109,12 @@ Read the [article](https://risczero.com/news/zeth-release).
 [proofs]: /terminology#validity-proof
 [prover]: /terminology#prover
 [publicly shared outputs]: /terminology#journal
+[segments]: /terminology#segment
 [verify]: /terminology#verify
 [zero-knowledge virtual machine]: /terminology#zero-knowledge-virtual-machine-zkvm
 [zkvm]: https://github.com/risc0/risc0#readme
 [zkVM Quick Start]: ./quickstart.md
 [zkVM example applications]: https://github.com/risc0/risc0/tree/v0.18.0/examples
```

Comment:
For `api/`, this link should point to `main`.
For `api_versioned_docs`, this should point to the corresponding version.

Reply:
```suggestion
[zkVM example applications]: https://github.com/risc0/risc0/tree/main/examples
```